registry/registry_test.go

```diff
@@ -77,50 +78,50 @@ func TestEndpoint(t *testing.T) {
 	assertInsecureIndex := func(index *registrytypes.IndexInfo) {
 		index.Secure = true
 		_, err := NewV1Endpoint(index, "", nil)
-		assertNotEqual(t, err, nil, index.Name+": Expected error for insecure index")
-		assertEqual(t, strings.Contains(err.Error(), "insecure-registry"), true, index.Name+": Expected insecure-registry  error for insecure index")
+		assert.Check(t, err != nil, index.Name+": Expected error for insecure index")
+		assert.ErrorContains(t, err, "insecure-registry", index.Name+": Expected insecure-registry  error for insecure index")
```

Comment:
Nice catch; I think those were added because previously, it would call `err.Error()` (and likely a linter complained about that). Let me remove those.

Reply:
fixed 👍